gangsta_chess/logic/piece/bishop.py

```diff
@@ -4,3 +4,11 @@
 class Bishop(Piece):
     def __init__(self, pos, team, Board):
         super().__init__(pos, team, Board)
+
+    def find_valid_moves(self):
```

Comment:
lets keep this and the other find_valid_moves in a feature branch until its working